### PR TITLE
fix: fixing isDisplayedInViewportSpec

### PR DESCRIPTION
**Pull request by ntatuani-cloudinary**
## Brief Summary of Changes
-

## Reviewer, Please Note:
-

## Testing (what was tested)
-

### CODE REVIEW

File: src/commons/BrowserUtils.ts [hunk 1]
```diff
@@ -219,9 +219,10 @@ export namespace BrowserUtils {
   /**
    * Return true if the selected DOM-element found by given selector is partially visible and within the viewport.
    */
-  export async function isDisplayedInViewport(selector: string): Promise<void> {
+  export async function isDisplayedInViewport(selector: string): Promise<boolean> {
     await Reporter.debug('Check if element displayed in viewport');
-    await tryBlock(async () => $(selector).isDisplayedInViewport(), '${selector} not found in the viewport');
+    return $(selector).isDisplayedInViewport();
+    // await tryBlock(async () => $(selector).isDisplayedInViewport(), '${selector} not found in the viewport');
```
Comment (FelixZilber) [inline src/commons/BrowserUtils.ts:225]: please remove

File: src/test/specs/isDisplayedInViewportSpec.ts [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-const selector = '#DragAndDropSpec';
+const existSelector = '#DragAndDropSpec';
+const notExistSelector = '#DragAndDropSpec123';
 
 describeCommon('isDisplayedInViewport', () => {
-  it('element displayed in viewport', async () => {
+  it('element is not displayed in viewport', async () => {
     await Reporter.step('check if element is displayed in viewport');
-    await BrowserUtils.isDisplayedInViewport(selector);
+    chai.expect(await BrowserUtils.isDisplayedInViewport(notExistSelector)).to.be.false;
```
Comment (FelixZilber) [inline src/test/specs/isDisplayedInViewportSpec.ts:11]: notExist is not the same as not in view port...

**Review by FelixZilber — CHANGES_REQUESTED** [review]
see comments inline

**Comment by FelixZilber** [comment]
@ntatuani-cloudinary any updates regarding this one?

**Comment by ntatuani-cloudinary** [comment]
testnow

**Review by FelixZilber — APPROVED** [review]

**Pull request merged**